tests/storage_test.go

```diff
@@ -431,7 +431,7 @@ var _ = Describe("Storage", func() {
 
 				By("Checking that the VirtualMachineInstance console has expected output")
 				expecter, err := tests.LoggedInCirrosExpecter(vmi)
-				Expect(err).To(BeNil())
+				Expect(err).To(BeNil(), "Cirros login successfully")
```

Comment:
To be clear, Expect messages are only ever displayed if the expected condition was not met, so something along the lines of "expected to be able to log in to cirros" might be more clear.

By the way, I think we generally try to express error tests specifically:

`Expect(err).ToNot(HaveOccurred())`

It just changes the test output from "expected <random error> to equal nil" to something along the lines of "we didn't expect an error, but we got one. here it is....".

Reply:
mh, about the expect message: isn't it convention that the message expresses what we expect? That's at least what @petrkotas does, and he suggested to add these messages. See e.g. https://github.com/kubevirt/kubevirt/pull/1531/files

Will change the err expectations, thanks for the heads up!